client/pom.xml

```diff
@@ -3,20 +3,20 @@
 
     <modelVersion>4.0.0</modelVersion>
     <artifactId>client</artifactId>
-    <version>1.0.0</version>
+    <version>1.0.1</version>
```

Comment:
If current app version is always the same like the parent, I prefer write down <version> in properties